makefiles/host/docker_entry.sh

```
@@ -2,24 +2,26 @@
+    if [ ! -f 'build/root.fs.loopback.img' ]; then
         truncate -s 4G build/root.fs.loopback.img
         mkfs.btrfs build/root.fs.loopback.img
     fi
-    losetup -d /dev/loop0
+    losetup -d /dev/loop0 2> /dev/null
     losetup /dev/loop0 build/root.fs.loopback.img
     mount /dev/loop0 build/root.fs
 fi
 
-make -f makefiles/host/main.mk -I makefiles/host -j $(nproc) $*
+make -f makefiles/host/main.mk -I makefiles/host -j "$(nproc)" $*
```

Shellcheck (0.10.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)